ml-agents/mlagents/trainers/ppo/trainer.py

```diff
@@ -229,6 +229,7 @@ def add_policy(
         if not isinstance(policy, NNPolicy):
             raise RuntimeError("Non-NNPolicy passed to PPOTrainer.add_policy()")
         self.policy = policy
+        self.policies[parsed_behavior_id.behavior_id] = policy
```

Comment:
I am against adding self.policies to all of the trainers. Having a dictionary of policies only makes sense for the ghost trainer, having both self.policies and self.policy seems super redundant to me.

Reply:
This was already done.  `self.policies` is defined in the base `Trainer` class.  This change is remedying the fact that the policies weren't actually assigned in `PPOTrainer` and `SACTrainer`.

I agree we should change it, but wanted to avoid further expanding the scope of this PR.

Reply:
😲

Reply:
@andrewcoh

Reply:
IMO we should just make it consistent - since `self.policies` is more general, we should just have it as the standard across trainers. That way all trainers can share the `get_policy` method.